Multisig/UI/Safe Management/SafeNameForm/SafeNameFormModel.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SafeNameFormModel.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 24.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class SafeNameFormModel: ObservableObject {
+
+    @Published
+    var enteredText: String = ""
+
+    @Published
+    var isValid: Bool?
+
+    var address: String
+
+    private var subscribers = Set<AnyCancellable>()
+
+    init(address: String) {
+        self.address = address
+        
+        $enteredText
+        .dropFirst()
+        .sink { value in
+            self.isValid = !value.isEmpty
```

Comment:
should we have "too long value" logic?

Reply:
I would add, actually.

Reply:
that would need more changes to the model. I would write it down for the future.